.gitignore

```diff
@@ -1,3 +1,6 @@
+# JetBrains project directory
+/.idea
```

Comment:
generally, this kind of thing would be in your user level `.gitignore` but I'm cool with it nonetheless

Reply:
As @dangreenisrael noted it's best to set this up globally.

```
# *nix
git config --global core.excludesfile ~/.gitignore

# Windows
git config --global core.excludesfile %USERPROFILE%\.gitignore
```

Reply:
Agreed - I've reverted this change.